core/packages/test/scripts/deploy-ethereum.sh

```diff
@@ -15,7 +15,8 @@ start_geth() {
             --rpc.allow-unprotected-txs --mine --miner.threads=1 \
             --miner.etherbase=0xBe68fC2d8249eb60bfCf0e71D5A0d2F2e292c4eD \
             --authrpc.addr="127.0.0.1" \
-            --http.addr="127.0.0.1" \
+            --http.addr="0.0.0.0" \
+            --http.corsdomain '*' \
```

Comment:
Interesting, do you know why this is necessary at this point? 🤔

Reply:
Ah, actually nothing to do with this PR. Sometimes I use `remixd` remote connect to the local network for debug purpose, adding this allow me to do that. 
 Since it is only a dev setup, I would assume should be fine but correct me if I'am wrong.

Reply:
Very cool! Please link to a tutorial or docs of how to set this up. I have been copy-pasting solidity back and forth into remix website for debugging. So It would be helpful to my workflow.

Reply:
Ok. Actually it's a websocket daemon connect web ide and I've added that in previous PR, just run `pnpm remixd` [here](https://github.com/Snowfork/snowbridge/blob/525126eabc92f2be27ce1b8ab7f580a5b6c01d02/core/package.json#L17) start the daemon and then connect to localhost as [following](https://remix-ide.readthedocs.io/en/latest/remixd.html)

I think other steps are same as what we use remix website as before, just compile code and connect to the local node with `External HTTP Provider`, you can deploy new contract or attach to an exist contract  and do some debug stuff then.

@alistair-singh Hope that is helpful.

Reply:
> Ah, actually nothing to do with this PR. Sometimes I use `remixd` remote connect to the local geth node for debug purpose, adding this allow me to do that. Since it is only a dev setup, I would assume should be fine but correct me if I'am wrong.

Awesome, should be fine!